### PR TITLE
Change jibSerialize to jib.serialize

### PR DESCRIPTION
**Pull request by loosebazooka**
part of #1967, take a look at where the warning is happening, open to suggestions on other ways to do this.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -533,16 +533,15 @@ com.google.api.client.http.level=CONFIG
 
 And then launch your build tool as follows:
 ```sh
-mvn -Djava.util.logging.config.file=path/to/log.properties -DjibSerialize=true -Djib.console=plain ...
+mvn -Djava.util.logging.config.file=path/to/log.properties -Djib.serialize=true ...
```
[the remark this answers is not in the document]
Comment (loosebazooka) [reply docs/faq.md]: oh oops, must've been lost in a rebase

**Comment by chanseokoh** [comment]
I think we should hold off merging this until right before we make the next release, since it updates the FAQ to use the new (not yet live) flag.

**Comment by loosebazooka** [comment]
> IMHO we shouldn't bother with the deprecation notice: it's a debugging property, not a production property. We allow seeing the executor which is what someone should do if they want control.

I think this is harder when we have to change the property `sendCredentialsOverHttp` to `jib.sendCredentialsOverHttp`, also I'm a little confused why these properties are separate from `jib-plugins-common/.../PropertyNames.java`

**Comment by loosebazooka** [comment]
> I think we should hold off merging this until right before we make the next release, since it updates the FAQ to use the new (not yet live) flag.

I can move the faq changes into the 1.6 PR: https://github.com/GoogleContainerTools/jib/pull/1938

**Comment by chanseokoh** [comment]
> I think this is harder when we have to change the property `sendCredentialsOverHttp` to `jib.sendCredentialsOverHttp`, also I'm a little confused why these properties are separate from `jib-plugins-common/.../PropertyNames.java`

Things in `PropertyNames` can apply only to our plugins. For example, `jib.console` makes sense only with the build loggers.

**Pull request merged**